client/src/App.jsx

```diff
@@ -1,35 +1,37 @@
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import Home from "./Pages/Home";
+import SongPage from "./Pages/SongPage";
+import ArtistPage from "./pages/ArtistPage";
 import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
-import "./App.css";
-
+const Router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Home />,
+  },
+  {
+    path: "/song",
+    element: <SongPage />,
+  },
+  {
+    path: "/Artist",
+    element: <ArtistPage />,
+  },
+]);
 function App() {
-  const [count, setCount] = useState(0);
-
+  const [showError, setShowError] = useState(false);
+  const handleClick = () => {
+    setShowError(true);
+  };
+  if (showError) {
+    throw new Error("Oops! Something went wrong");
+  }
   return (
-    <>
+    <div>
+      <RouterProvider router={Router} />
       <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
+        <button onClick={handleClick}>Click me</button>
```

Comment:
we don't need this button or the handleClick error function. you can remove them